src/ExposureHistory/ExposureDetail.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FunctionComponent } from "react"
+import { TouchableOpacity, View, StyleSheet, Linking } from "react-native"
+import { RouteProp, useNavigation } from "@react-navigation/native"
+import { useTranslation } from "react-i18next"
+import { SvgXml } from "react-native-svg"
+import env from "react-native-config"
+
+import { ExposureHistoryStackParamList } from "../navigation"
+import { RTLEnabledText } from "../components/RTLEnabledText"
+import { Screens, useStatusBarEffect } from "../navigation"
+import { DateTimeUtils } from "../utils"
+
+import { Colors, Iconography, Outlines, Spacing, Typography } from "../styles"
+import { Icons } from "../assets"
+
+const {
+  GAEN_AUTHORITY_NAME: healthAuthorityName,
+  AUTHORITY_ADVICE_URL,
+  DISPLAY_SELF_ASSESSMENT,
+} = env
+
+interface ExposureDetailProps {
+  route: RouteProp<ExposureHistoryStackParamList, "ExposureDetail">
+}
+
+const ExposureDetail: FunctionComponent<ExposureDetailProps> = ({ route }) => {
```

Comment:
Would it be similar to passing the props here to use the `useRoute` hook annotated with the type we created already? Like [here] it is probably ok but since we were using hooks already might worth checking it out

[here]: https://reactnavigation.org/docs/typescript/#annotating-useroute